clients/admin-ui/src/features/privacy-requests/privacy-requests.slice.ts

```diff
@@ -501,31 +501,60 @@ export const {
 } = privacyRequestApi;
 
 export type CORSOrigins = Pick<SecurityApplicationConfig, "cors_origins">;
-const EMPTY_CORS_DOMAINS: CORSOrigins = {
-  cors_origins: [],
+export type CORSOriginsSettings = {
+  configSet: Pick<
+    SecurityApplicationConfig,
+    "cors_origins" | "cors_origin_regex"
+  >;
+  apiSet: CORSOrigins;
 };
-export const selectCORSOrigins = () =>
+export const selectCORSOrigins: (state: RootState) => CORSOriginsSettings =
   createSelector(
     [
       (state) => state,
       privacyRequestApi.endpoints.getConfigurationSettings.select({
         api_set: true,
       }),
+      privacyRequestApi.endpoints.getConfigurationSettings.select({
+        api_set: false,
+      }),
     ],
-    (_, { data }) => {
-      const hasCorsOrigins = narrow(
+    (_, { data: apiSetConfigSettings }, { data: configSetConfigSettings }) => {
+      // TODO: holy *** add a comment for this
```

Comment:
Should there be something else here instead of the TODO?

Reply:
HAHA thank you.

Reply:
Honestly, on reflection... the runtime type check that `narrow` is performing here is just creating some unnecessary complexity here. I'm going to remove it so that the logic is simpler to read.

Reply:
Cleaned this up and left some helpful code comments. Thanks for the reminder